### PR TITLE
feat(docs): add 'special properties' section

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: static/docs/src.html [hunk 1]
```diff
@@ -367,6 +367,10 @@ <h2>HTML Attributes</h2>
       <h2>Custom Elements</h2>
       <section data-include="rs-changelog"></section>
     </section>
+    <section>
+      <h2>Special properties</h2>
+      <section data-include="document.respec"></section>
```
[the remark this answers is not in the document]
Comment (sidvishnoi) [reply static/docs/src.html:372]: Added custom IDs in wiki (`document.respec.toHTML` instead of `document-respec-toHTML`)

**Review by sidvishnoi — APPROVED** [review]

**Comment by sidvishnoi** [comment]
There's a ReSpec regression on our docs about license, which will prevent rendered doc from getting updated on server. Can you send a PR/commit for that?
image

File: static/docs/src.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (sidvishnoi) [reply static/docs/src.html:372]: Argh. Our custom ID regex doesn't support dots. Should we support it or stick to hyphens?
https://github.com/w3c/respec/blob/a0d26d954c981059c2a94c208a1b9c17a0b027c5/src/core/markdown.js#L99

[the remark this answers is not in the document]
Comment (marcoscaceres) [reply static/docs/src.html:372]: I don't mind... whatever is less work.

**Comment by marcoscaceres** [comment]
https://github.com/w3c/respec-web-services/pull/245

**Pull request merged**